kani-compiler/src/args.rs

```diff
@@ -87,7 +87,7 @@ pub struct Arguments {
     /// Option name used to select which backend to use.
     #[clap(long = "backend", default_value_t = BackendOption::CProver)]
     pub backend: BackendOption,
-    /// Print the final LLBC file to stdout. This requires `-Zaeneas`.
+    /// Print the final LLBC file to stdout.
```

Comment:
Why did you remove this comment?

Reply:
My understanding is that those requires comments should be in the kani-driver help, and not in the Kani compiler one and the kani-driver option already mentions the requirement:

https://github.com/model-checking/kani/pull/3630/files#diff-6f15b74109fec7be3b13bdc2f8c01481d82be82f413c39a7f54d53f1990c6454R280

None of the other Kani compiler options that require an unstable flag (e.g. `--check_coverage`, `--ub-check`) mention the unstable flag requirement in their help message.